OrbitGl/ImGuiOrbit.h

```diff
@@ -40,6 +40,9 @@ IMGUI_API void Orbit_ImGui_KeyCallback(GlCanvas* a_Canvas, int key, bool down);
 IMGUI_API void Orbit_ImGui_CharCallback(GlCanvas* a_Canvas, unsigned int c);
 
 void SetupImGuiStyle(bool bStyleDark_, float alpha_);
+bool LoadTextureFromFile(const char* filename, uint32_t* out_texture,
+                         int* out_width, int* out_height);
+uint32_t LoadTextureFromFile(const char* filename);
```

Comment:
Document possible return values? (In particular, 0 means error here. Something like: "Returns the texture id generated by OpenGL. In case of an error, returns 0.")